app/javascript/src/LoginPage/LoginPageWrapper.jsx

```diff
@@ -37,6 +37,7 @@ type Props = {
   providerSessionsPath: string,
   redirectUrl: string,
   show3scaleLoginForm: boolean,
+  masterAccount: boolean,
```

Comment:
I think it's not ideal to mention "master account". We should make it a more generic feature:
```suggestion
  disablePasswordReset: boolean,
```